calculator.js

```diff
@@ -1,28 +1,58 @@
-
-const exampleAdditionInput = {
+const inputSample = {
   num1: 3,
-  num2: 5,
-  operation: 'add',
+  num2: 6,
+  operation: '+',
 }
-// '3 + 5 = 8'
 
-const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+const calculateUserInput = function (error, input) {
+  const operation = input.operation;
+
+  if (isNaN(input.num1)) {
+    console.log(`${input.num1} is not a number`)
+    return
+  }
+  const num_1 = Number(input.num1)
+
+  if (isNaN(input.num2)) {
+    console.log(`${input.num2} is not a number`)
+    return
+  }
+  const num_2 = Number(input.num2)
```

Comment:
don't forget your semicolons!
```suggestion
  if (isNaN(input.num1)) {
    console.log(`${input.num1} is not a number`);
    return;
  }
  const num_1 = Number(input.num1);

  if (isNaN(input.num2)) {
    console.log(`${input.num2} is not a number`);
    return;
  }
  const num_2 = Number(input.num2);
```